build/Targets/Roslyn.Toolsets.Xunit.targets

```diff
@@ -2,6 +2,10 @@
 <!-- Copyright (c)  Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information. -->
 <Project ToolsVersion="14.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
```

Comment:
In another file you removed the expanded `<Project>` syntax. This one remains.

Reply:
:arrow_right: The ones removed were required as part of the switch from PowerShell object syntax to `XmlNode.SelectSingleNode`. The alternative is managing a namespace table in the PowerShell code.